sdk_container/src/third_party/coreos-overlay/app-crypt/clevis/clevis-19-r1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Automated Encryption Framework"
+HOMEPAGE="https://github.com/latchset/clevis"
+SRC_URI="https://github.com/latchset/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
This KEYWORS prevents the builds from starting.

```
!!! All ebuilds that could satisfy "clevis" for /build/amd64-usr/ have been masked.
!!! One of the following masked packages is required to complete your request:
- app-crypt/clevis-19-r1::coreos (masked by: ~amd64 keyword)
```

You would probably want to add `=app-crypt/clevis-19-r1 ~amd64` in `sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords`, or replace the KEYWORDS above with `KEYWORDS="amd64"`.